process/labs-process.md

```diff
@@ -0,0 +1,121 @@
+
+* Submitting a [new lab proposal] for review by the [OpenSSF TAC].
+* Ensuring all commits are properly signed-off to avoid issues related to [DCO].
+* Notifying the TAC if the lab needs to be suspended or archived.
+
+Labs projects are also highly encouraged to engage with the [existing
+Technical Initiatives] (working groups, projects or SIGs) in OpenSSF to build
+their community and find a potential pathway towards acceptance as an OpenSSF
+project.
+
+## Archiving
+
+The TAC will periodically check on the activity of labs. Labs that have been
+inactive for an extended period (6+ months), or are explicitly suspended by
+the maintainers, will be moved into the [Archived
+stage](templates/LAB_NAME_archived_stage.md).
+
+Archived lab repositories are not actively maintained and will be marked as
+"archived" (read-only) on GitHub. They can be reactivated if there is interest
+in resuming work on a project.
+
+## New Lab Proposal Process
+
+1. Fork the `<TBD>` repo.
+
+2. Fill out the [proposal template](templates/LAB_NAME_lab_stage.md)
+   and save it into the `labs` subdirectory under the name of your lab,
+   such as `coolnewproject_lab.md`.
+   <br/>
+> [!TIP]
+> It is expected that your lab repository on GitHub will have the same
+> name as the proposal, so keep that in mind when submitting your proposal.
+
+3. In the proposal template, there is an entry for sponsor(s). Although this
+   is not required, proposers are encouraged to seek a sponsor in the OpenSSF
+   community who can help them create ties with the rest of the community
+   and review the proposal to make sure it is novel and aligned with the
+   [OpenSSF mission].
+   <br/>
+   To find sponsors:
+     1. use your connections to existing projects and ask maintainers,
+     2. engage with [existing Technical Initiatives] (working groups, projects
+	    or SIGs) with affinities to the proposed lab and pitch the project in
+		their meetings or [Slack channels](https://slack.openssf.org/). It's
+		good to have the template already filled out when you reach out.
+   <br>
+> [!IMPORTANT]
+> Lab sponsors may but are not required to actively participate in
+> the lab once the proposal has been reviewed and accepted.
+
+4. Commit your changes with proper sign-off. This means that your commit
+   log message must contain a line that looks like the following one,
+   with your actual name and email address:
+
+   `Signed-off-by: John Doe <john.doe@example.com>`
+
+   Adding the `-s` flag to your `git commit` command will add that line
+   automatically. You can also add it manually as part of your commit
+   log message or add it afterwards with `git commit --amend -s`.
+
+5. Submit a Pull Request to the `<TBD>` repo.
+
+The [OpenSSF TAC] will then review your proposal. Like sponsors, TAC members may
+but are not required to participate in ongoing work like contributing or
+reviewing code in the lab.
+
+### Transferring an existing repository
```

Comment:
Would transferring existing repositories over to the labs org require a full license and IP review as for "full" projects coming into the OpenSSF? That could create a significant overhead and we may want to trade-off the intention of making labs a very accessible path into the OpenSSF space and putting the necessary guardrails in place.

Reply:
I don't think transferring an existing repo to the labs org should require a full license and IP review, as long as they have one of the expected licenses and DCO. The full license and IP review becomes part of the transition requirement to sandbox stage. I can clarify this in the text.

CC @Naomi-Wash @riaankleinhans for confirmation on this.

Reply:
I've updated the text with a note about this, please let me know what you think.